<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureDatabaseWidget</class>
 <widget class="QWidget" name="TextureDatabaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>file name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pFileName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pTransparencyColor">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>transp. color:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pChoseFile">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>chose file</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="m_pView">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="dragMode">
    <enum>QGraphicsView::ScrollHandDrag</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pChoseColor">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>chose color</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pPath">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
